packages/rspack/src/schemas/plugins/LoaderOptionsPlugin.check.js

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
How would we generate this file if `scheme.json` changes?

Reply:
For the short term, we will not change the plugin. We can port the `scheme generation` in the next quarter.